compiler/damlc/tests/daml-test-files/InterfaceChoiceCollision2.daml

```diff
@@ -0,0 +1,14 @@
+-- Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+-- @SINCE-LF-FEATURE DAML_INTERFACE
+module InterfaceChoiceCollision2 where
```

Comment:
This test confirm there is not collision check for requiring interfaces.

Reply:
> This test confirms there is not collision check for requiring interfaces.

Can you add this as a comment in the test?

Reply:
> Can you add this as a comment in the test?

I will literally drop the collision check in the next PR, I assumed it was not necessary.